recipes/assimp/3.3.1/conanfile.py

```diff
@@ -0,0 +1,128 @@
+import os
+from conans import ConanFile, CMake, tools
+
+
+class Assimp(ConanFile):
+    name = "assimp"
```

Comment:
```suggestion
    name = "assimp"
    version = "3.3.1"
```
As you are writing a very specific version, you should declare it in your recipe. Only avoid it when using the generic folder `all`, or any generic version.